html/index.html.in

```diff
@@ -203,6 +203,7 @@
                                         </div>
                                         <div class="row mt-2">
                                             <div class="col-12 col-md-6 d-flex d-sm-block flex-wrap otherTranslationOptions">
+                                                <button type="button" id = "clearText" class="btn btn-secondary mt-3"><i class="fa fa-trash"></i> &nbsp; Clear</button>
```

Comment:
Remove spaces around =, and note how none of the other buttons use `&nbsp;` but instead wrap the text in a translatable element.